Services/ContentExport.php

```diff
@@ -169,6 +183,16 @@ public function generateTable(ContentType $contentType, $format)
                 }
             }
 
+            foreach ($this->registry->getConverters($contentType) as $converter) {
+                if (isset($columnNames)) {
```

Comment:
Is there any way that the $columnNames are not set ? If so then maybe you should fix that problem instead of working around it.